dbus/dbus_libgio.go

```diff
@@ -0,0 +1,43 @@
+// Copyright 2020 Northern.tech AS
+//
+//    Licensed under the Apache License, Version 2.0 (the "License");
+//    you may not use this file except in compliance with the License.
+//    You may obtain a copy of the License at
+//
+//        http://www.apache.org/licenses/LICENSE-2.0
+//
+//    Unless required by applicable law or agreed to in writing, software
+//    distributed under the License is distributed on an "AS IS" BASIS,
+//    WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+//    See the License for the specific language governing permissions and
+//    limitations under the License.
+// +build !nodbus,cgo
+
+package dbus
+
+// #cgo pkg-config: gio-2.0
+// #include <gio/gio.h>
+import "C"
+
+type DBusAPILibGio struct {
+	d DBusAPI
```

Comment:
Does this actually need to be here? It is implicitly fulfilled, right?

Reply:
Right!